### PR TITLE
[IMP] Improved imports of modules

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: oca_dependencies.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+1)	account_payment_mode
+2)	account_payment_partner
+3)	account_payment_sale
+4)	connector
+5)	connector_base_product
+6)	connector_ecommerce
+7)	product_multi_category
+8)	sale_automatic_workflow
+9)	sale_automatic_workflow_payment_mode
+10)	sale_exception
```
Comment (JayVora-SerpentCS) [inline oca_dependencies.txt]: This is not the format of oca_dependencies.txt! 
See https://github.com/OCA/sale-workflow/blob/11.0/oca_dependencies.txt

Comment (JayVora-SerpentCS) [reply oca_dependencies.txt]: mention repo, not the module name!

File: shopwareerpconnect/invoice.py [hunk 2]
```diff
@@ -25,7 +25,7 @@
 from openerp.addons.connector.queue.job import job, related_action
 from openerp.addons.connector.unit.synchronizer import Exporter
 from openerp.addons.connector.event import on_record_create
-from openerp.addons.connector_ecommerce.event import (on_invoice_paid,
+from openerp.addons.connector_ecommerce.models.event import (on_invoice_paid,
                                                       on_invoice_validated)
 from openerp.addons.connector.exception import IDMissingInBackend
```
Comment (JayVora-SerpentCS) [inline shopwareerpconnect/invoice.py:30]: should be odoo. not openerp. everywhere! This is v9 onwards

File: shopwareerpconnect/payment_method.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 
 class PaymentMethod(models.Model):
-    _inherit = "payment.method"
+    _inherit = "account.payment.mode"
```
Comment (JayVora-SerpentCS) [inline shopwareerpconnect/payment_method.py]: is the model renamed as such ? Refer here the proof plz

File: oca_dependencies.txt (hunk 1 above)
Comment (ghost) [reply oca_dependencies.txt]: Yes of course, I will add repositories instead of modules

File: shopwareerpconnect/invoice.py (hunk 2 above)
Comment (bhaskar622) [reply shopwareerpconnect/invoice.py]: Yes in from version 9 it is odoo not openerp.

Comment (ghost) [reply shopwareerpconnect/invoice.py:30]: No sir it is "openerp" in version 9 from 10 it changes to "odoo", I have checked it in base's modules and few others they all have "openerp" in import line.

File: shopwareerpconnect/payment_method.py (hunk 3 above)
Comment (ghost) [reply shopwareerpconnect/payment_method.py]: It replaces the object "payment.method"
    of the module "sale_payment_method" of OCA/sale-workflow from v8.

This is written in "account_payment_mode" module from OCA/bank-account in v9 under "account.payment.mode" model.

And the fields that are defined in "payment.method" in v8 are mostly same in v9 except their model name is replaced with "account.payment.mode"

https://github.com/OCA/bank-payment/blob/9.0/account_payment_mode/models/account_payment_mode.py

**Pull request merged**